src/styles/themes/_theme-dark.scss

```diff
@@ -2,7 +2,8 @@ $colors: (
   primary: #bb99ff,
   contrast: #ffffff,
   primary-text: #ffffff,
-  background-primary: #000000
+  background-primary: #000000,
+  disabled-primary: grey
```

Comment:
Considering that this is used for text can we duplicate this for text? So `disabled-text` for text and `disabled-primary` for elements (the input element).